.github/.jira_sync_config.yaml

```
@@ -0,0 +1,16 @@
+settings:
+  jira_project_key: "OBC"
+  status_mapping:
+    opened: Untriaged
+    closed: done
+    not_planned: rejected
+
+  components:
+    - opentelemetry-collector-rock
```

In mimir-rock we [omit](https://github.com/canonical/mimir-rock/blob/1abdc8bcd1f955c5543f0248cab9cca89191a876/.github/.jira_sync_config.yaml#L9) the `-rock` suffix.
```suggestion
    - opentelemetry-collector
```